android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1107,6 +1112,8 @@ public boolean onRequestPermissionsResult(int requestCode, String[] permissions,
                 if (grantResults[i] == PackageManager.PERMISSION_GRANTED) {
                   if (permissions[i].equals(Manifest.permission.CAMERA)) {
                     grantedPermissions.add(PermissionRequest.RESOURCE_VIDEO_CAPTURE);
+                  } else if (permissions[i].equals(Manifest.permission.RECORD_AUDIO)) {
```

Comment:
tbh in order for audio in the webview to work we need more than this permissions alone.

We need these, maybe we can remove `MODIFY_AUDIO_SETTINGS`, haven't checked:
```
    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
    <uses-permission android:name="android.permission.AUDIO_CAPTURE" />
```

Reply:
Is `MODIFY_AUDIO_SETTINGS ` required too?